### PR TITLE
add composer.json

### PR DESCRIPTION
**Pull request by dawehner**
https://www.drupal.org/node/2331489

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "drupal/entity_embed",
+  "description": "Allows entities to be rendered using a text format.",
+  "type": "drupal-module",
```
Comment (davereid) [inline composer.json:4]: If we're using the custom 'type' information, why aren't we also requiring composer/installers as per https://getcomposer.org/doc/faqs/how-do-i-install-a-package-to-a-custom-path-for-my-framework.md ?

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "license": "GPL-2.0+",
+  "minimum-stability": "dev",
```
Comment (davereid) [inline composer.json:6]: Don't think we need this line?

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  "require": { }
```
Comment (davereid) [inline composer.json:7]: Do we need to have an empty require list, or can this just be excluded?

File: composer.json (hunk 1 above)
Comment (dawehner) [inline composer.json:4]: oh mh, I haven't thought of requiring it. we do actually have that composer already part of our main composer.json file, see https://blog.erdfisch.de/node/412

File: composer.json (hunk 2 above)
Comment (dawehner) [inline composer.json:6]: mh, github doesn't tell me which one you exactly meant. the license or the minimum-stablity?

File: composer.json (hunk 3 above)
Comment (dawehner) [inline composer.json:7]: Technically you should at least add the required php version.

File: composer.json (hunk 2 above)
Comment (davereid) [inline composer.json:6]: minimum-stability - seems only required if we had requirements, but we don't (and won't) have any

File: composer.json (hunk 3 above)
Comment (davereid) [inline composer.json:7]: We don't need any additional requirement other than the version required by Drupal core, so I don't understand.

**Comment by ericduran** [comment]
I incorporated this PR with -> https://github.com/drupal-media/entity_embed/pull/94

I brought in this commit as is (https://github.com/ericduran/entity_embed/commit/af1418905672fb022d4c4ac60d75ef45ea2f4291) and made my changes on top of it. To make sure the history and committer is correct. :+1:

**Comment by cs-shadow** [comment]
Closing this PR in favor of #94.

**Comment by davereid** [comment]
I still had open questions about certain things about this composer.json file though.

**Comment by cs-shadow** [comment]
@davereid Maybe we can move the discussion to the new PR. Feel free to reopen this PR if you like that better.

**Comment by ericduran** [comment]
@davereid I address all your concern on #94.

**Pull request closed without merging**